CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/),
+and this project adheres to [Semantic Versioning](https://semver.org/).
+
+## Unreleased
```

Comment:
why "Unreleased" if you intend to tag it as v1.0.0?

Reply:
Well I wanted to ask first but yes I see your point. I will also put a date in the header but I wanted to do that when I would actually tag the release in github...